src/gatsby/gatsby-browser.js

```diff
@@ -5,14 +5,17 @@ import "bootstrap/dist/js/bootstrap.bundle"
 import "@fortawesome/fontawesome-free/css/all.min.css"
 import "./src/styles/style.scss"
 
-export const onRouteUpdate = ({ location, prevLocation }) => {
-  if (typeof window.snowplow === 'function') {
-    window.snowplow("trackPageView");
-    // refresh snowplow link click tracking to pick up new links
-    window.snowplow("refreshLinkClickTracking");
-  }
-  sessionStorage.setItem("prevPath", prevLocation ? prevLocation.pathname : null);
```

Comment:
is `prevPath` in sessionStorage not needed anymore?

Reply:
Good catch! Accidentally deleted it 😅